index.js

```diff
@@ -37,14 +57,14 @@ class CacheBookend extends BookendInterface {
      * @return {Promise}        Resolves to a string that represents the commmand to execute
      */
     getTeardownCommand(o) {
-        const eventCache = hoek.reach(o.job, 'permutations.0.cache.event');
+        const cache = hoek.reach(o.job, 'permutations.0.cache');
 
-        if (eventCache && eventCache.length > 0) {
-            const eventMap = eventCache.map(item =>
-                      `store-cli set ${item} --type=cache --scope=event || true`
-                  );
+        if (cache) {
```

Comment:
Would be worth adding `cache.length > 0` check here just in case, but shouldn't be a problem as is, i guess

Reply:
This one is an object get from config parser. It's either there or not. The check for length is inside the getCacheCommands method.